resources-generator/src/main/kotlin/dev/icerock/gradle/generator/platform/apple/SetupAppleUtils.kt

```diff
@@ -159,6 +159,15 @@ internal fun registerCopyFrameworkResourcesToAppTask(
         ?: emptyMap()
 
     val configName: String = (configMap[configuration]?.name ?: configuration).lowercase()
+    // Map any configuration to standard with two types: "Debug/Release" builds
+    val clearConfigurationName: String = when {
+        configName.matches(".*debug.*".toRegex()) -> "debug"
+        configName.matches(".*release.*".toRegex()) -> "release"
```

Comment:
it's still not usable for all cases.
i think we should give properties to map xcode configuration to gradle configuration.
just like `xcodeConfigurationToNativeBuildType` in cocoapods kotlin plugin.

and i see that  on 155 line we already read `xcodeConfigurationToNativeBuildType` property. so if we already read it - we incorrectly use it and this should be fixed.

for cases when in xcode used custom configurations but not used cocoapods gradle plugin we still should provide mapping just like `xcodeConfigurationToNativeBuildType` to give required control for users